src/playground/sample-data-ingestion/generator.py

```diff
@@ -0,0 +1,187 @@
+from faker import Faker
```

Comment:
generator.py, could you please specify a little bit for the name? is it data/monitor/detector/alert/plugin/dashboards generator?